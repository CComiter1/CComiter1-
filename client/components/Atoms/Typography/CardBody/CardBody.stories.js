import React from 'react'
import { storiesOf } from '@storybook/react'

import CardBody from './CardBody'

storiesOf('Atoms/Typography/CardBody', module)
  .add('short text', () => <CardBody>Hello World</CardBody>)
  .add('long text', () => (
    <CardBody>
      asdfjasdfj;alkjfkajs;dlfkajsdlkfj;ljeih
      oasdpjfio jqwpioejfiwe jpio fjpiowjas;ldf jpioaskld fjpa
      as ;ldfjkalsd fj;askld fj;laksdj f;akld fja;klsd

      as;kld fj;asdlfja;lsdk fja
      s;klad jfa;ls
    </CardBody>
  ))
