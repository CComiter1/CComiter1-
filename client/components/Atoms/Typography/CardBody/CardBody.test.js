import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import CardBody from './CardBody'

describe('CardBody', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <CardBody>TEST 123</CardBody>
    ).toJSON()
    expect(component).toMatchSnapshot()
  })
})
