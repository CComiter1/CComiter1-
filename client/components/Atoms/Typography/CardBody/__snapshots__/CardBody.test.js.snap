// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBody renders correctly 1`] = `
.c0 {
  font-family: 'Roboto',sans-serif;
  margin: 0;
}

<p
  className="c0"
>
  TEST 123
</p>
`;
